src/plan/gencopy/mod.rs

```diff
@@ -5,4 +5,18 @@ pub(super) mod global;
 pub(super) mod mutator;
 
 pub use self::global::GenCopy;
+
+pub const NO_BARRIER: bool = false;
```

Comment:
Are those 3 constants only used for you to run our experiments? If my understanding is correct, they change the behavior of gen copy, and they should not be set to `true` for a normal user. If that's the case, I would suggest removing them from the code. It is more about the methodology you used for the experiments.

Reply:
I suggest we keep these flags. These flags provide a way to run GenCopy without any barriers, or with a completely different barrier. With these flags, one can directly clone mmtk-core and openjdk binding, and start doing barrier overhead measurements without further changes to the GenCopy GC, which is quite convenient.  We also need this feature later for barrier fast path debugging.

Reply:
Okay. If we have to keep them, I suggest adding some more comments:
1. For where the constants are defined, explain how to config those constants to measure overhead for different components
2. For where those constants are used, be very clear that the constant is only used for special methodology for measuring overhead of different components. The reason why I suggest removing them is those conditional checks are quite disruptive when I tried to read the code, as some essential code will be skipped based on the constant - which is confusing.

Reply:
Done.